File: sdk/async-scala/src/main/scala/io/swagger/client/model/BuyerCreditCard.scala
package io.swagger.client.model

import org.joda.time.DateTime
import java.util.UUID


case class BuyerCreditCard (
  Token: Option[String],
CardType: Option[String],
PartialAccountNumber: Option[String],
CardholderName: Option[String],
ExpirationDate: Option[Date],
xp: Option[Any]
)

File: sdk/javascript-closure-angular/API/Client/BuyerCreditCard.js
goog.provide('API.Client.BuyerCreditCard');

/**
 * @record
 */
API.Client.BuyerCreditCard = function() {}

/**
 * @type {!string}
 * @export
 */
API.Client.BuyerCreditCard.prototype.token;

/**
 * @type {!string}
 * @export
 */
API.Client.BuyerCreditCard.prototype.cardType;

/**
 * @type {!string}
 * @export
 */
API.Client.BuyerCreditCard.prototype.partialAccountNumber;

/**
 * @type {!string}
 * @export
 */
API.Client.BuyerCreditCard.prototype.cardholderName;

/**
 * @type {!API.Client.date}
 * @export
 */
API.Client.BuyerCreditCard.prototype.expirationDate;

/**
 * @type {!API.Client.Object}
 * @export
 */
API.Client.BuyerCreditCard.prototype.xp;

